src/main/java/lotto/domain/Ball.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+public class Ball{
+    private static final String NUMBER_MATCHES = "-?[0-9]+";
+    public static final int MINIMUM_NUMBER = 1;
+    public static final int MAXIMUM_NUMBER = 45;
+    private static final String ERROR_ONLY_NUMBER = "숫자를 입력해주세요!";
+    private static final String ERROR_LOTTO_NUMBER = "로또 숫자가 아닙니다";
+
+    private final int number;
+
+    public Ball(String number) {
+        checkValidValue(number);
+        this.number = Integer.parseInt(number);
+    }
+
+    private void checkValidValue(final String value) {
```

Comment:
요런 경우에 저는 주로 validate라는 동사를 사용합니다. 어휘를 풍부하게 만들어드릴 조언이라 생각해주세요 😬

```suggestion
    private void validateNumber(final String value) {
```